test/Libraries/Microsoft.Extensions.AI.AotCompatibility.TestApp/Microsoft.Extensions.AI.AotCompatibility.TestApp.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>net9.0</TargetFrameworks>
+    <PublishAot>true</PublishAot>
+    <TrimmerSingleWarn>false</TrimmerSingleWarn>
+    <SelfContained>true</SelfContained>
```

Comment:
```suggestion
```

I don't believe we need this. I've seen other repos copy it, but I'm not sure why.

Reply:
Your blog linked to https://github.com/open-telemetry/opentelemetry-dotnet/blob/f2c225519db42e97c978bbb87b765da3204de860/test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj#L6-L35 as an example to follow. I just copied it from there.